plugins/DevicesDetection/Columns/BrowserName.php

```diff
@@ -17,12 +19,42 @@ class BrowserName extends Base
 {
     protected $columnName = 'config_browser_name';
     protected $columnType = 'VARCHAR(10) NULL';
-    protected $segmentName = 'browserCode';
+    protected $segmentName = 'browserName';
     protected $nameSingular = 'DevicesDetection_ColumnBrowser';
     protected $namePlural = 'DevicesDetection_Browsers';
-    protected $acceptValues = 'FF, IE, CH, SF, OP, etc.';
+    protected $acceptValues = 'FireFox, Internet Explorer, Chrome, Safari, Opera etc.';
     protected $type = self::TYPE_TEXT;
 
+    public function __construct()
+    {
+        $this->sqlFilterValue = function ($val) {
+            $browsers = Browser::getAvailableBrowsers();
+            $result   = array_search($val, $browsers);
```

Comment:
Would it be a good idea if we made the segment case insensitive?

Reply:
When choosing a suggest value that shouldn't matter. But shouldn't hurt to compare them lower case. Will change them